README.md

```diff
@@ -23,16 +23,16 @@ For DevOps or SRE teams this charm will make operating Synapse simple and
 straightforward through Juju's clean interface. It will allow easy deployment
 into multiple environments for testing of changes.
 
+## Get started
+
+To begin, refer to the [Getting Started](https://charmhub.io/synapse/docs/tutorial-getting-started)
+tutorial for step-by-step instructions.
+
```

Comment:
Are there any basic operations (e.g. actions) or integrations worth mentioning for this charm?

Reply:
My bad, I didnt notice that "Basic Operations" wasnt optional, I'll add some!